ChatRPG/Areas/Identity/RevalidatingIdentityAuthenticationStateProvider.cs

```diff
@@ -47,22 +47,20 @@ public class RevalidatingIdentityAuthenticationStateProvider<TUser>
         }
     }
 
-    private async Task<bool> ValidateSecurityStampAsync(UserManager<TUser> userManager, ClaimsPrincipal principal)
+    private async Task<bool> ValidateSecurityStampAsync(UserManager<TUser?> userManager, ClaimsPrincipal principal)
```

Comment:
Build warning til linje 50:
`Warning: The type 'TUser?' cannot be used as type parameter 'TUser' in the generic type or method 'UserManager<TUser>'. Nullability of type argument 'TUser?' doesn't match 'class' constraint.`